### DISCUSSION TITLE
Can't get started with @vanilla-extract/css on a basic Hello World

### DISCUSSION THREAD

**Discussion by kyplerdeloitte** [vanilla-extract-css/vanilla-extract#1136] [Q&A]
I am unable to assign the styles (background: red) I created with "import {styles} from vanilla-extract/css" to a simple div with the text "Hello World". I tried using different bundler integrations (vite, webpack, gatsby and bagel) but I keep getting this same error:

![image](https://github.com/vanilla-extract-css/vanilla-extract/assets/113994597/6efc3e46-3888-47d5-a128-091d4143bb95)

This is the repo
https://github.com/kyplerdeloitte/hello-world

**Comment by rothsandro** [comment]
The integration depends your setup, e.g. the Vite plugin will not work for your create-react-app setup. CRA is dead anyway, so I recommend that you create a new project with Vite or Next and use the official plugins. If you really want to keep create-react-app, follow the instructions described here: https://sandroroth.com/blog/vanilla-extract-cra
